content/blog/optimizing-tinacms-with-nextjs-and-vercel-data-cache.mdx

```diff
@@ -15,11 +15,17 @@ next: ''
 
 **Update 19/3/2025:** Added a video to demo how the Vercel Data Cache could affect your site and how to tweak Next.js so that you can better use the cache.
 
-<Youtube embedSrc="https://youtube.com/embed/gz9Lxlj5RDU" />
+**Update 31/3/2025:** Added a couple images to add additional information on enabling the Vercel Data Cache and possible issues.
+
+<Youtube embedSrc="https://youtube.com/embed/gz9Lxlj5RDU" caption="TinaCMS - Using nextjs and the vercel data cache" minutes="6" />
 
 In December 2024, with the release of `tinacms@2.5.2` and `@tinacms/cli@1.7.0`, TinaCMS transitioned to using Node.js’s native fetch function, eliminating the need for the fetch-ponyfill dependency. This change reduced dependencies, enhancing both security and performance. It also allowed us to support the **Vercel Data Cache**.
 
-When integrating TinaCMS with Next.js and deploying on Vercel, it’s essential to understand Vercel’s Data Cache mechanism since it operates seamlessly. This caching system stores API responses, reducing redundant requests and improving page load times. However, Vercel caches TinaCloud’s API responses for an extended period by default, which can lead to stale content. To manage revalidation and ensure content freshness, consider the following approaches:
+When integrating TinaCMS with Next.js and deploying on Vercel, it’s essential to understand Vercel’s Data Cache mechanism since it operates seamlessly. This caching system stores API responses, reducing redundant requests and improving page load times. However, Vercel caches TinaCloud’s API responses for an extended period by default, which can lead to stale content.
+
+![](/img/blog/optimizing-tinacms-with-nextjs-and-vercel-data-cache/one-year-caching.png "One of our sites using Tina experienced late content updates due to 1 year default caching.")
```

Comment:
Is this image already in the repo? or was it meant to be added as part of this PR?